### PR TITLE
Use terms for branding.

### PR DESCRIPTION
**Pull request by mrstegeman**
References #2326

### CODE REVIEW

**Comment by flodolo** [comment]
Here's one problem: localization has already started via GitHub, which means some locales already have these strings translated.

In the future, contributions should only happen via Pontoon, and to make similar changes you'll need to update the string ID. 

For this PR, you can either do that (change IDs), or you need to go through all languages and replace all occurrences of `WebThings Gateway` with the corresponding term.

**Comment by mrstegeman** [comment]
@flodolo I already did the latter and replaced the term in all of the translations.

**Comment by flodolo** [comment]
> For this PR, you can either do that (change IDs), or you need to go through all languages and replace all occurrences of `WebThings Gateway` with the corresponding term.

Sorry, tunnel vision at 6AM. That's exactly what you did and that's perfect ;-)

File: static/fluent/en-US/main.ftl [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+-mozilla-iot-brand = Mozilla IoT
+
 # Main Title
-webthings-gateway = WebThings Gateway
+webthings-gateway = { -webthings-gateway-brand }
 
 # Wordmark
 wordmark =
-  .alt = Mozilla IoT
+  .alt = { -mozilla-iot-brand }
```
Comment (benfrancis) [inline static/fluent/en-US/main.ftl]: Where in the UI is this being used?

Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: It's the `alt` text for the wordmark image.

Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: I suppose we should just be using the WebThings Gateway brand.

Comment (benfrancis) [reply static/fluent/en-US/main.ftl]: Yeah I think this should have been changed when the logo was changed

Comment (benfrancis) [reply static/fluent/en-US/main.ftl]: (probably my bad)

Comment (mrstegeman) [reply static/fluent/en-US/main.ftl]: Fixed.

File: static/fluent/sr/main.ftl [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+##
+## They cannot be:
+## - Transliterated.
+## - Translated.
+##
+## Declension should be avoided where possible.
+##
+## Reference: https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark
+
+-webthings-gateway-brand = { -webthings-gateway-brand }
```
Comment (benfrancis) [inline static/fluent/sr/main.ftl]: Should this be `-webthings-gateway-brand = WebThings Gateway` ?

Comment (mrstegeman) [reply static/fluent/sr/main.ftl]: Yes, great catch! Fixed.

**Pull request merged**